client/src/cordova/apple/xcode/ios/Outline/Classes/AppKitBridge/AppKitController.swift

```diff
@@ -20,9 +20,30 @@ class AppKitController: NSObject {
 
     override public required init() {
         super.init()
-
-        // Indicates that the application is an ordinary app that appears in the Dock and may have a user interface.
```

Comment:
Please restore. The explanation of the policy is still valid. Perhaps explain this changes later. Explain when and where.